server/handlers/genomic_annotations.go

```diff
@@ -1 +1,155 @@
 package handlers
+
+import (
+	"errors"
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/ldsec/medco-connector/restapi/models"
+	"github.com/ldsec/medco-connector/restapi/server/operations/genomic_annotations"
+	utilserver "github.com/ldsec/medco-connector/util/server"
+	"github.com/sirupsen/logrus"
+	"strings"
+)
+
+var genomicAnnotationsTypes = []string{"variant_name", "protein_change", "hugo_gene_symbol"}
+
+func MedCoGenomicAnnotationsGetValuesHandler(params genomic_annotations.GetValuesParams, principal *models.User) middleware.Responder {
+
+	err := utilserver.DBConnection.Ping()
+	if err != nil {
+		logrus.Error("Impossible to connect to DB: " + err.Error())
+		return genomic_annotations.NewGetValuesDefault(500).WithPayload(&genomic_annotations.GetValuesDefaultBody{
+			Message: "Impossible to connect to DB " + err.Error(),
+		})
+	}
+
+	var query string
+
+	query, err = buildGetValuesQuery(params)
+	if err != nil {
+		logrus.Error("Query execution error " + err.Error())
+		return genomic_annotations.NewGetValuesDefault(500).WithPayload(&genomic_annotations.GetValuesDefaultBody{
+			Message: "Query execution error: " + err.Error(),
+		})
+	}
+
+	//escaping * characters
```

Comment:
Ok, here we have to agree on the behaviour we want to have. In itself the * character is not a dangerous one that needs to be escaped for security reasons. But it is a wildcard. So if  user enters it as part of a string to search for and we do not escape it, the list of returned results  will be constructed according to the sql regexp rules. Thing is that there are annotation values that contain the character *, so I thought it would make sense to disable complex pattern matching and just goes with the basic one avoiding ambiguous wildcards. But if you prefer I can leave the * character as it is and leave to the users the task to escape it if they know what they want, it is up to us.

Reply:
So this relates as well with the other comments. Let's handle it at the same time that we take care of the loader.